language-command.php

```diff
@@ -11,27 +11,15 @@
 }
 
 WP_CLI::add_command( 'language core', 'Core_Language_Command', array(
-	'before_invoke' => function() {
-		if ( \WP_CLI\Utils\wp_version_compare( '4.0', '<' ) ) {
-			WP_CLI::error( 'Requires WordPress 4.0 or greater.' );
-		}
-	})
+	'before_invoke' => wpcli_language_check_requirements())
```

Comment:
You're executing the function immediately here, which is not the intended behavior. This will interrupt other commands then.

Together with the change into a closure below, we can just pass that closure and WP-CLI will execute it at the right time.
```suggestion
	'before_invoke' => $wpcli_language_check_requirements)
```

Reply:
Grr.. doing too many things at the same time ;-) Fixed now.